primitives/src/v1.rs

```diff
@@ -114,7 +106,7 @@ pub struct CandidateDescriptor<H = Hash> {
 	pub relay_parent: H,
 	/// The collator's sr25519 public key.
 	pub collator: CollatorId,
-	/// The blake2-256 hash of the validation data. This is extra data derived from
+	/// The blake2-256 hash of the persisted validation data. This is extra data derived from
```

Comment:
A persisted validation data hash? Or the hash of the whlole `ValidationData`? Field name should probably be `persisted_validation_data_hash`.